deeposlandia/dataset.py

```diff
@@ -101,18 +102,24 @@ def add_label(self, label_id, label_name, color, is_evaluate,
         is_evaluate : bool
         category : str
             String designing the category of the dataset label
-        aggregate : list (optional)
+        aggregate_label_ids : list (optional)
             List of label ids aggregated by the current label_id
+        contained_labels : list
+            List of raw labels aggregated by the current label
         """
         if label_id in self.label_info:
             utils.logger.error("Label {} already stored into the label set.".format(label_id))
             return None
         self.label_info.append({"name": label_name,
                                 "id": label_id,
-                                "category": (label_name if category is None
+                                "category": (label_name
```

Comment:
the python ternary operator is often written in a single line

`val = rhs if predicate else other_value`

Reply:
I agree with you. What if the line is too long? Here I can define two variables in dedicated lines before the `append` statement to keep the code easily readable.

Reply:
As you wish.